Submodels/MTSOL_call.py

```
@@ -1,20 +1,613 @@
+        Returns
+        -------
+        None
+        """
+
+        # Construct output file name
+        output_file = f'{file_name}.{self.analysis_name}'
+
+        # Read all files in the directory
+        file_pattern = f'MTSOL_output_files/{file_name}.{self.analysis_name}*'
+        files = glob.glob(file_pattern)
+        content = [open(file).readlines() for file in files]
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)